src/da65/handler.c

```diff
@@ -508,9 +546,24 @@ void OH_DirectIndirectLongX (const OpcDesc* D attribute ((unused)))
 
 
 
+static void impl_StackRelativeIndirectY (const char *sp, const OpcDesc* D attribute ((unused)))
```

Comment:
For the sake of consistency with the rest of this file I'd prefer to have this function removed and the OneLine() directly put into the two OH_...() functions.